acceptance/openstack/loadbalancer/v2/quotas_test.go

```diff
@@ -45,16 +59,21 @@ func TestQuotasUpdate(t *testing.T) {
 		log.Fatal("Original and New Loadbalancer Quotas are the same")
 	}
 
-	// Restore original quotas.
-	restoredQuotas, err := quotas.Update(client, os.Getenv("OS_PROJECT_NAME"), quotas.UpdateOpts{
+	var restoredQuotaUpdate = quotas.UpdateOpts{
 		Loadbalancer:  &originalQuotas.Loadbalancer,
 		Listener:      &originalQuotas.Listener,
 		Member:        &originalQuotas.Member,
 		Pool:          &originalQuotas.Pool,
 		Healthmonitor: &originalQuotas.Healthmonitor,
-		L7Policy:      &originalQuotas.L7Policy,
-		L7Rule:        &originalQuotas.L7Rule,
-	}).Extract()
+	}
+	// L7 parameters are only supported in microversion v2.19 introduced in victoria
+	if clients.IsReleasesAbove(t, "stable/ussuri") {
```

Comment:
Unrelated to your patch but it looks like the implementation of L7 parameters is buggy, they should be pointers according to https://github.com/gophercloud/gophercloud/blob/master/docs/MICROVERSIONS.md#new-response-fields.

Same for tls version above.

Reply:
could you file an issue please?